docs/standards/hcs-16.md

```diff
@@ -0,0 +1,528 @@
+
+<!-- > **NOTE**   If any step fails to reach `T` acknowledgements within `TTL` (default = 7 days), the initiator **SHOULD** recycle the partial resources and mark the Flora as **Aborted**. -->
+
+### Internal Flora Topics
+
+| Topic                            | Purpose                                                                                                                                                                                                                                 | Required | Memo Prefix           |
+| -------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | --------------------- |
+| **Communication Topic (CTopic)** | Human / agent chat, off‑chain URL exchange, policy proposals, general communications. Future standards are planned to define task coordination.                                                                                         | ✅       | `hcs-22:${floraId}:0` |
+| **Transaction Topic (TTopic)**   | Broadcast of pre‑signed ScheduledTxn IDs, token association proposals, contract calls.                                                                                                                                                  | ✅       | `hcs-22:${floraId}:1` |
+| **State Topic (STopic)**         | Flora shared state that is needed for the Flora's purpose. Example: Periodic SHA‑256 hashes of shared state, vector clocks, membership attestations. This Topic is required for valid Floras even if not utilized intiially by members. | ✅       | `hcs-22:${floraId}:2` |
+
+All three topics **SHOULD** share the exact same `adminKey` that is the same as the Flora's account key. `submitKey` **SHOULD** always be 1/M keylist threshold to allow each member to submit indepently.
+
+> **NOTE** HIP-991 custom fees can be added to require HBAR or a custom fungible token / NFT to submit messages on these topics. This should be agreed upon before Flora creation by member petals.
+
+### Flora Memo Structure
+
+The flora account memo follows a standardized format to indicate where the profile data is stored:
+
+```
+hcs-11:<protocol_reference>
+```
+
+Where:
+
+- `hcs-11` is the protocol identifier
+- `<protocol_reference>` can be either:
+  - A [Hashgraph Resource Locator (HRL)](../definitions.md#hashgraph-resource-locator) for HCS protocols
+  - Other URI formats for non-HCS protocols (IPFS, Arweave, HTTPS)
+
+Examples of valid hcs-22 account memo formats:
+
+```
+# HRL references (HCS protocols)
+hcs-11:hcs://1/0.0.8768762
+hcs-11:hcs://2/0.0.8768762
+hcs-11:hcs://7/0.0.8768762
+
+# Non-HCS protocol references
+hcs-11:ipfs://QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX
+hcs-11:ar://TQGxHPLpUcH7NG6rUYkzEnwD8_WqYQNPIoX5-0OoRXA
+```
+
+### HCS-11 extended for HCS-16 Root Profile Schema
```

Comment:
NOTE: 

1. We need to migrate this into HCS-11 and reference it via a link to keep all profile specifications together. 

2. We should consider using JSON-schema for HCS-11 versioning